### ISSUE TITLE
View localState in Apollo Dev Tools (cache tab)

### ISSUE THREAD

**Issue by Natas0007** [gabo2192/patio-comida#1]
Hey there!  I'm using your apollo/cache.js design of using typePolicies and a reactive var to handle local state.  This seems to be working great except that I'm unable to view the state in Apollo Dev Tools.  I'm using Apollo Client v3.1.4.  I'm able to write the contents of localState to the console just fine when using the useQuery hook to execute the query...but nothing in the cache tab of the dev tools.  Have you experienced this issue?  I'm experimenting with moving all local state into Apollo from Redux for one of my projects, but if I can't consistently view the local state without writing to the console, I can't justify making the switch.  Thank you for your time!

**Comment by gabo2192** [comment]
Yes, Apollo Dev Tools isn't working properly at the moment, at least not properly. I heard that there should be upcoming changes to that.
